### PR TITLE
1.22.4 Release Notes

### PR DESCRIPTION
**Pull request by zirain**
xref: https://github.com/istio/release-builder/pull/1924

### CODE REVIEW

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (wilsonwu) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:37]:
```suggestion
- **Fixed** an issue that can trigger a deadlock when `discoverySelectors` (configured in `MeshConfig`) and a namespace,
  which has an `Ingress` object or a `gateway.networking.k8s.io` `Gateway` object, moves from being selected to unselected.
```

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:30]:
```suggestion
- **Fixed** handling of a a failure adding a pod to the dataplane where the pod was still added to ipset.
  ([Issue #52218](https://github.com/istio/istio/issues/52218))
```

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:27]:
```suggestion
- **Fixed** an omission in ztunnel helm charts which resulted in some Kubernetes resources being created without labels.
```

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:36]:
```suggestion
- **Fixed** an issue that can trigger a deadlock when `discoverySelectors` (configured in `MeshConfig`) and a namespace,
  which has an `Ingress` object or a Kubernetes `Gateway` object, would move from being selected to unselected.
```

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:40]:
```suggestion
- **Removed** writing the experimental field `GatewayClass.status.supportedFeatures`, as it was unstable in the API.
```

**Review by dhawton — CHANGES_REQUESTED** [review]

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:16]:
```suggestion
- **Fixed** an issue where the `VirtualMachine` `WorkloadEntry` locality label was missing during auto-registration.
```

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (dhawton) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:19]:
```suggestion
- **Fixed** matching multiple service VIPs in `ServiceEntry`.
```

File: content/en/news/releases/1.22.x/announcing-1.22.4/index.md
Comment (craigbox) [inline content/en/news/releases/1.22.x/announcing-1.22.4/index.md:19]:
```suggestion
- **Fixed** a case where listeners were missing for addresses beyond the first in a `ServiceEntry`.
```

Comment (craigbox) [reply content/en/news/releases/1.22.x/announcing-1.22.4/index.md:19]: this is a nit you can easily ignore

**Review by craigbox — APPROVED** [review]

**Review by dhawton — APPROVED** [review]

**Comment by dhawton** [comment]
Good to merge when you're ready. The cherrypick will need approval/hold until publish.

**Pull request merged**
